packages/polymath-issuer/src/actions/sto.js

```diff
@@ -248,11 +249,18 @@ export const configureSTO = (config: STOConfig, type: string) => async (
         const stoModuleConfig = {
           ...config.data,
         };
-
         const tokenBalance = await PolyToken.balanceOf(token.address);
         const titles = ['Deploying And Scheduling'];
 
         if (tokenBalance.lt(setupCost)) {
+          if (!hasEnoughBalance) {
```

Comment:
It looks like this shouldn't be required if we are already checking in the above line for `tokenBalance.tl(setupCost)`.

Also, it is safer to get the balance right before we do the action as it is done in line `252`

Reply:
Nevermind, this is checking for the Security Token's balance